### DISCUSSION TITLE
Empirical Data -> Model is unclear

### DISCUSSION THREAD

**Discussion by frankelavsky** [Myndex/SAPC-APCA#41] [Q&A] [answered]
I am unsure which parts of APCA are actually influenced by which empirical studies. It would be really helpful from a scientific standpoint to know this information and have it readily available either in the repo, documented in the code as comments, or main site for APCA.

I found a list of papers here, akin to a bibliography:
https://www.w3.org/WAI/GL/task-forces/silver/wiki/Visual_Contrast_of_Text_Subgroup/Resources#Key_References

But these don't seem to point to specific parts of APCA. It is very difficult to figure out which studies influenced which parts of the model without reading every study and examining the model. In academic science (for example), typically models are rigorously documented. Perhaps progress towards this work can be used to develop the methodology section of a research paper? (As a way to maximize the labor involved in this project, since I assume there will be a paper eventually.)

In particular, knowing which empirical studies influenced which parts of APCA will:
- Help generate trust in the model's rigor
- Provide opportunities for scientists/practitioners to improve parts of the model or contribute
- Help scientists/practitioners recognize potential gaps or unaddressed areas (such as text outlines/etc)

This addition to the growing body of materials will really help drive consensus towards adopting APCA (not just consensus among the academic community but political and practitioner spaces as well).

**Comment by Myndex** [comment]
Hi Frank, this is an ongoing project with ongoing studies.

We are conducting our own studies, and have been since 2019 — COVID interfered with some of the data collection unfortunately.

## _Available Materials:_
Here is an early whitepaper that is more explanatory: https://www.myndex.com/WEB/WCAG_CE17polarity

That follows an earlier model, and things have been well refined since then. Some of the earlier experiments are listed: https://www.myndex.com/WEB/Perception

But what you might find interesting is the "research mode" at the SAPC research site: http://www.myndex.com/SAPC/

Click on **_Research Mode,_** and there are some technology demonstrators the more fully present the methods.

### _Your Papers Please...._
As to your other question, there are multiple papers in the works, along with certain IP protections for IP that has not been released yet. Again, this is an ongoing project, and I am being as open as possible given the need for IP protection for some of the unreleased materials.

Thank you for your interest, I will happilly answer further questions,

Andy


_Andrew Somers_     
_W3 AGWG Invited Expert_     
_Myndex Senior Color Scientist_     
_Inventor of SAPC and APCA_     

## _THE REVOLUTION WILL BE READABLE_ <sup>™</sup>

**Comment by Myndex** [comment, marked as answer]
Hi @frankelavsky  to add: The building of the indexed bibliography DB (probably my least favorite part LOL) is moving forward, but also I just did a search and there are over 2000 .pdf papers collected over the last nearly three years.

The folder of "definitely in the bib" that I am organizing has 117 papers to cite... 

> _In academic science (for example), typically models are rigorously documented_

Yes, and that is revealed when the papers are submitted to referees and published. I know the degree ongoing research is or is not _open_ varies. I'm used to never at all open corporate R&D, under NDA, (OMG when I was at a certain toy co, where it's swell, the NDA was like 78 pages). Published articles in trade magazines (film/TV related), which are a little different than an academic paper in that you get _paid_ for writing for a trade pub. 

But for "academic" papers?? You gotta pay _THEM??_ Pay to play science. Biggest scam in the world. And people wonder why I'm not so interested in that process? Especially when the **evidence of efficacy is prima facia** and I have the basic experiments and math **_all public facing_** so anyone that wants to can see and recreate.

So that's my point — let me know as you climb the learning curve any questions. I'm not eager to _spend $2000_ to submit an open access paper just so an anonymous peer review board can look it over. Maybe crowd fund LOL.

Thank you!

Andy

**Comment by Myndex** [comment]
Moving this to discussions
